infrastructure/dpladm/env-repo-template/standard/.lagoon.yml

```diff
@@ -82,4 +82,4 @@ backup-retention:
     hourly: 0
 # https://docs.lagoon.sh/lagoon/using-lagoon-the-basics/lagoon-yml#backup-schedule
 backup-schedule:
-  production: M H(22-2) * * *
+  production: M 12,19 * * *
```

Comment:
May be worth considering keeping the jitter (c.f. [what H does in cron syntax](https://stackoverflow.com/a/47302727/1080564)) here?

Reply:
yes, might be a good idea aswell. It does jitter on the minute though